pkgs/build-support/fetchsvn/default.nix

```diff
@@ -32,13 +38,16 @@ else
 stdenvNoCC.mkDerivation {
   name = name_;
   builder = ./builder.sh;
-  nativeBuildInputs = [ subversion glibcLocales ];
+  nativeBuildInputs = [ subversion glibcLocales ]
+    ++ stdenvNoCC.lib.optional sshSupport openssh;
```

Comment:
Is this necessary, given the next line?

Reply:
Probably not, but I sort of like to explicitly put down deps like that anyways, in case there is or will be a setup hook or something.

Reply:
I'd already merged to master in #82130 So I'll mere this as I have more commits on top, but happy to change this on both branches later if you like.